aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -220,54 +220,180 @@ For example, the `Certificates:Default` certificate can be specified as:
 }
 ```
 
-#### ConfigurationLoader
+### Configure client certificates in appsettings.json
```

Comment:
This feels like a new topic.  I'd probably put it after all the ways to configure the server certificate.

Reply:
Probably together with "Configure client certificates in code"

Reply:
The proposed plan in #21117 was to group all the using-config sections together and the using-code sections together.

Reply:
Ping @JamesNK

Reply:
I don't think there is a perfect answer.

We have different ways to configure: URLs, config, code
We have different things to configure: address, HTTPS, protocol

We have to pick one.